ultravox/model/ultravox_config_test.py

```diff
@@ -25,8 +38,13 @@ def test_can_load_release(model_id: str):
     config_reloaded_diff = ultravox_config.UltravoxConfig(
         **config_from_dict.to_diff_dict()
     )
-    assert config_reloaded.to_dict() == orig_config.to_dict()
-    assert config_reloaded_diff.to_dict() == orig_config.to_dict()
+    assert (
+        exclude_key(config_reloaded.to_dict(), keys_to_ignore) == orig_config.to_dict()
+    )
```

Comment:
This rule is not gonna work in the future when these keys are added to `orig_config`. Here's a slightly different suggestion:
```suggestion
    assert (
        config_reloaded.to_dict() == {**{k: None for k in keys_to_ignore}, **orig_config.to_dict()}
    )
```